sunflower-land/sunflower-land
# [FIX] Hot Module Replacement

# Description

Updates the Hot Module Replacement so that it doesn't do a full page reload.

## How to Test

1. Open up any modal, and modify a style. 
2. Check vite terminal and ensure an `hmr update` is performed
3. Ensure page doesn't reload

![hmr](https://github.com/user-attachments/assets/ecd1db10-8a4e-433b-ba88-4670f719b01a)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

localhost

# Checklist:
